### ISSUE TITLE
pyorerun Issue - no display

### ISSUE THREAD

**Issue by aaa34169** [pyomeca/pyorerun#29]
Hello.

i installed pyorerun. ( i may miss the information, but pyorerun is python 3.11 compatible only)
i wanted to run the running example. 
prr windows poped up but did not display anything. 
i paste my screen 
![prr-runningIssue](https://github.com/pyomeca/pyorerun/assets/3098117/cd857c18-5c21-4023-8514-abadaa23f39f)

regards

Fabien

**Comment by Ipuch** [comment]
Thanks for trying, let me know if the update works.  
- update to last version of the package pyorerun (download again or follow the main branch if you are using git)
- `pip install .` on your current env should make the work

the software rerun is eveloving quickly, it's hard to follow, I modifed the readme to stick to 0.16.1

**Comment by aaa34169** [comment]
Thanks for your prompt reply. 
I installed in developer mode to consider any updates. 
i ll keep you in touch if that works

**Comment by Ipuch** [comment]
I think this is fine. I'm closing the issue as a consider this resolved. Let me know if you have troubles with the new version.
